CONTRIBUTING.md

```diff
@@ -239,7 +244,24 @@ sudo cp /home/$(whoami)/go/bin/MailHog /usr/local/bin/mailhog
 mailhog
 ```
 
-To access your MailHog inbox, open your browser and navigate to [http://localhost:8025](http://localhost:8025). For any other questions related to MailHog or for instructions on custom configurations, check out the [MailHog](https://github.com/mailhog/MailHog) repository.
+#### Using MailHog
+
+Once you have installed MailHog and started it running you need to open your MailHog inbox in your browser, open a new tab or window and navigate to [http://localhost:8025](http://localhost:8025). 
+You should now see a screen like below:
+
+![MailHog Screenshot 1](https://github.com/johnkennedy9147/Resources/blob/master/mailhogImages/1.jpg?raw=1)
```

Comment:
Can you add images in `/docs` as done in case of other files previously. And also repeat the same for other images